src/main/java/service/WritableToCsvFile.java

```diff
@@ -0,0 +1,7 @@
+package service;
+
+import java.io.File;
+
+public interface WritableToCsvFile {
+    void writeToCsvFile(File file);
+}
```

Comment:
Use more general names
```suggestion
public interface WriterService {
    void writeToFile(String data, String outputPath);
}
```